neo4j/README.md

```diff
@@ -50,7 +55,6 @@ neo4j does not include any events.
 
 ## Troubleshooting
 
-
 Need help? Contact [Datadog support][7].
```

Comment:
This should be Neo4j support email

Reply:
Thanks. I was out of the office last week.
Just made the requested changes.